docs/model-dev-guide/dtrain/dtrain-introduction.rst

```diff
@@ -8,93 +8,93 @@
  How Determined Distributed Training Works
 *******************************************
 
-Distributed training in Determined utilizes data-parallelism. Data-parallelism for deep-learning
-consists of a set of workers, where each worker is assigned to a unique compute accelerator such as
-a GPU or a TPU. Each worker maintains a copy of the model parameters (weights that are being
-trained), which is synchronized across all the workers at the start of training.
+Determined employs data parallelism in its approach to distributed training. Data parallelism for
+deep learning consists of a set of workers, where each worker is assigned to a unique compute
+accelerator such as a GPU or a TPU. Each worker maintains a copy of the model parameters (weights
+that are being trained), which is synchronized across all the workers at the start of training.
 
 After initialization is completed, distributed training in Determined follows a loop where:
 
 #. Every worker performs a forward and backward pass on a unique mini-batch of data.
```

Comment:
Could consider a diagram here? Would help break up the text.

Reply:
Agree. I'll gather some requirements for the diagram.

Reply:
created a Jira ticket and a mockup, likely a separate PR

Reply:
i'll set this to resolved since the graphic design is a separate ticket